src/mesher.cc

```diff
@@ -48,15 +48,17 @@ void build_rigidbody(const glm::mat4 &m, btCollisionShape *shape, btRigidBody **
         btRigidBody::btRigidBodyConstructionInfo
             ci(0, ms, shape, btVector3(0, 0, 0));
         *rb = new btRigidBody(ci);
+        (*rb)->setSleepingThresholds(0.1f, 0.1f);
+        (*rb)->setCollisionFlags((*rb)->getCollisionFlags() | btCollisionObject::CF_STATIC_OBJECT);
```

Comment:
Describe what you're doing with collision flags in the commit message?

Reply:
done